### PR TITLE
Scheduler now respects default timeout and result_ttl

### PR DESCRIPTION
**Pull request by simone6021**
As per subject, solves #292.

Thanks for the awesome work!

### CODE REVIEW

**Comment by selwin** [comment]
Before I merge this in, could you please improve the test coverage by modifying https://github.com/rq/django-rq/blob/master/.travis.yml#L36 to install `rq-scheduler` prior to running tests?

**Comment by simone6021** [comment]
Here you are.  

Should document the possibility to set a custom scheduler class in settings and maybe add it to the scheduler commad.

File: django_rq/.kdev4/django_rq.kdev4 [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[Buildset]
```
Comment (selwin) [inline django_rq/.kdev4/django_rq.kdev4]: This file shouldn't be committed. Could you please remove this file from the repo?

File: django_rq/django_rq.kdev4 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[Project]
```
Comment (selwin) [inline django_rq/django_rq.kdev4]: This file should also be removed.

**Comment by simone6021** [comment]
This should solve also #278

**Comment by selwin** [comment]
Thanks!

**Pull request merged**